spec/rubocop/magic_comment_spec.rb

```diff
@@ -103,23 +125,29 @@
 
   include_examples(
     'magic comment',
-    '# -*- encoding: ASCII-8BIT; frozen_string_literal: true -*-',
+    '# -*- encoding: ASCII-8BIT; frozen_string_literal: true; ' \
+    'shareable_constant_value: literal -*-',
```

Comment:
This had to be split into 2 lines because of line length, but it makes the magic comment less readable. 🤔